cartridges/int_turnto_core_v5/cartridge/templates/default/turnto/htmlHeadJS.isml

```diff
@@ -59,15 +66,14 @@ NOTE: The template only features base functionality out of the box, in order to
 			}
 		};
 
-	<isset name="baseSiteVer" value="${Resource.msg('global.version.number','version', Resource.msg('revisioninfo.revisionnumber','revisioninfo','unknown version'))}" scope="page"/>
 	var extensionVersion = {
-		architecture: "${Resource.msg('turnTo.version.architecture','turnTo','') + ' ' + baseSiteVer}",
-		sfccPlatform: "${Resource.msg('turnTo.version.sfccPlatform','turnTo','')}",
-		turnToCartridge: "${Resource.msg('turnTo.version.turnToCartridge','turnTo','')}",
+		architecture: "${Resource.msg('turnTo.version.architecture','turnTo','')}",
+		sfccPlatform: "${Resource.msg('global.version.number','version', Resource.msg('revisioninfo.revisionnumber','revisioninfo','unknown version'))}",
+		turnToCartridge: "${Resource.msg('turnTo.version.turnToCartridge','turnTo','')}"
 	};
-
-	if('${pageID}' == 'pdp-page') {
-		var turnToConfig = {
+    var turnToConfig;
```

Comment:
Just a suggestion, since you are already moving the `turnToConfig` variable up.  What if you just set the shared properties here.
```
var turnToConfig = {
    locale: "${request.getLocale()}",
    extensionVersion: extensionVersion,
    sso: ssoConfig
};
```

Then set `pageId` and any other page specific properties below.  For example,
```
    turnToConfig.pageId = "pdp-page";
    turnToConfig.sku = "${product_sku}";
```

Reply:
done